pages/en/contrib/doc-contrib.md

```diff
@@ -25,13 +25,13 @@ Follow these steps:
 1. Click on **Edit this page** (top of page) to fork the repo.
 1. Edit the page or pages as needed.
 1. Commit to your fork/branch.
-1. Open a PR for your changes.  If there is an associated issue, please reference it in your PR.
+1. Open a PR for your changes.  If there is an associated issue, please reference it in your PR. The tutorial [Submitting a pull request to LoopBack 4](https://loopback.io/doc/en/lb4/submitting_a_pr.html) contains expert and beginner instructions to help a contributor with the pull request flow.
 
 {% include important.html content="When you submit your PR, be sure to agree to the contributor license agreement; see below.
 " %}
 
 We attempt to review and merge PRs as soon as possible; in general, we'll try to do so within 24 hours during the business week.  Please allow longer over weekends or holidays.
-
+                
```

Comment:
The extra spaces should be removed.

Reply:
done

Reply:
There's still some extra spaces (the darker green):
<img width="336" alt="screen shot 2019-02-27 at 9 08 36 am" src="https://user-images.githubusercontent.com/42985749/53496089-48487600-3a6f-11e9-957f-3a4a554ce81a.png">